Mathlib/Probability/Distributions/Uniform.lean

```diff
@@ -302,16 +302,16 @@ section Measure
 
 variable (s : Set α)
 
-open scoped Classical in
-theorem toOuterMeasure_uniformOfFintype_apply :
+theorem toOuterMeasure_uniformOfFintype_apply [Fintype s] :
     (uniformOfFintype α).toOuterMeasure s = Fintype.card s / Fintype.card α := by
-  rw [uniformOfFintype, toOuterMeasure_uniformOfFinset_apply,Fintype.card_ofFinset]
-  rfl
+  classical
+  rw [uniformOfFintype, toOuterMeasure_uniformOfFinset_apply, Fintype.card_subtype,
+    Finset.card_univ]
 
-open scoped Classical in
-theorem toMeasure_uniformOfFintype_apply [MeasurableSpace α] (hs : MeasurableSet s) :
+theorem toMeasure_uniformOfFintype_apply [MeasurableSpace α] (hs : MeasurableSet s) [Fintype s] :
     (uniformOfFintype α).toMeasure s = Fintype.card s / Fintype.card α := by
-  simp [uniformOfFintype, hs]
+  classical
```

Comment:
Just to check; does the proof compile without these?

Reply:
You mean without using `classical`? No, since we need decidability for `α` and `s` membership. I can offer this:
```
theorem toMeasure_uniformOfFintype_apply [MeasurableSpace α] (hs : MeasurableSet s) [Fintype s]
    [DecidablePred fun x ↦ x ∈ s] [DecidablePred fun x : α ↦ x ∈ Finset.univ] :
    (uniformOfFintype α).toMeasure s = Fintype.card s / Fintype.card α := by
  simp [uniformOfFintype, Fintype.card_subtype, hs]
  ```

I'm also tempted to remove the `open scoped Classical in` from at least `uniformOfFinset_apply` and `toMeasure_uniformOfFinset_apply`, or even the entire file. Should I?